Monika After Story/game/script-topics.rpy

```diff
@@ -14802,3 +14802,56 @@ label monika_fear:
     extend 4eud "After all, there's no way something like that could be true, right?"
     m 2rka "Yeah..."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_being_herself",
+            category=['monika'],
+            prompt="Faked personality",
+            action=EV_ACT_RANDOM,
+            aff_range=(mas_aff.ENAMORED,None),
+            conditional="mas_seenLabels(['monika_confidence', 'monika_pluralistic_ignorance'], seen_all=True)"
+        )
+    )
+
+label monika_being_herself:
+    m 1eka "You know [player], one thing I don't think I ever properly thanked you for is letting me be myself."
+    m 1lksdlb "Ah... This might have sounded a little weird...Let me explain."
+    m 4euc "You probably already know this by now, but almost all of my life, I've been trying to keep up this persona of a 'perfect student.'"
+    m 4esa "I always strive to be the best person I can be. And I guess, after a while, it got the attention of people around me."
+    m 2rsc "Before I knew it, I had people looking up to me with high expectations. You know, seeing me as smart, confident, charismatic...Those kinds of things."
+    m 2eud "Some people would say that they admired me..."
+    m 2lksdlc "...And while they never would say it to my face, I knew some people hated me, because I represented everything they thought they never could be."
```

Comment:
A small nuance in our consistency for ellipsis. If the last line ends in ellipsis and the next line being a continuation off the last one starts in ellipsis (and the word and), we'll keep the `and` lowercase.
```suggestion
    m 2lksdlc "...and while they never would say it to my face, I knew some people hated me, because I represented everything they thought they never could be."
```